mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxSessionMapperFactoryTest.java

```diff
@@ -62,7 +63,8 @@ public void beforeMethod() {
     public void testCreateMessageMapper() throws Exception {
         LOG.info("createMessageMapper");
         MailboxSession session = null;
-        CassandraMailboxSessionMapperFactory instance = new CassandraMailboxSessionMapperFactory(null, null, null, null);
+        DefaultMessageId.Factory messageIdFactory = new DefaultMessageId.Factory();
```

Comment:
let's face it : the tests I had to change don't have any value .... I just didn't delete them because I'm doing something else.